LISISO/commonfunctions.sh

```diff
@@ -247,8 +255,9 @@ function installbuildrpm()
 	if [ "$kmodrpm" != "" ] && [ "$msrpm" != ""  ];
 	then
 		echo "Installing the Linux Integration Services for Microsoft Hyper-V..."
-		rpm -ivh $kmodrpm $msrpm
+		rpm -ivh $kmodrpm $msrpm 2> ${ERR_LOG_FILE}
```

Comment:
Can't we filter out the warnings right here?
Something like rpm -ivh $kmodrpm $msrpm | grep -v "mlx5_ib".
Not understood why need to send to file and then cat the file.

Reply:
That will filter out the message appearing on stdout. Warning are thrown in stderr (fd:2). Hence capturing stderr and then filtering it out.

Reply:
This should work. Re-directing stderr to stdout and filtering out.